runtimes/pythonrt/ak_runner.py

```diff
@@ -236,6 +238,16 @@ def send_response(self, value):
     def extract_response(self, message):
         data = message['payload']['value']
         return pickle.loads(b64decode(data))
+    
+    def send_error(self, error, traceback):
+        message = {
```

Comment:
unrelated to this PR. could we replace message (and payload) by serializable class or similar?

Reply:
The ideal will be to have a schema for the messages and then generate code both for Go & Python (like protobuf).
But, I don't want to add any external dependency in Python (e.g. protobuf).
Opened ENG-821